src/app/enums/enums.ts

```diff
@@ -0,0 +1,8 @@
+export enum EnglishLevels {
```

Comment:
rename file to english-levels.enum.ts or something like that

Reply:
done